src/JWT.ts

```diff
@@ -129,8 +129,8 @@ export const SUPPORTED_PUBLIC_KEY_TYPES: PublicKeyTypes = {
      */
     'EcdsaPublicKeySecp256k1',
   ],
-  Ed25519: ['ED25519SignatureVerification', 'Ed25519VerificationKey2018'],
-  EdDSA: ['ED25519SignatureVerification', 'Ed25519VerificationKey2018'],
+  Ed25519: ['ED25519SignatureVerification', 'Ed25519VerificationKey2018', 'Ed25519VerificationKey2020', 'JsonWebKey2020'],
```

Comment:
linter is complaining here
```suggestion
  Ed25519: [
    'ED25519SignatureVerification',
    'Ed25519VerificationKey2018',
    'Ed25519VerificationKey2020',
    'JsonWebKey2020',
  ],
```